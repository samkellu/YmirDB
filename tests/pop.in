pop a
set a
pop a
set b 12
set a 123 44 b
get a
pop a
get a
pop a
get a
pop a
get a
pop a
get a
get b
bye
