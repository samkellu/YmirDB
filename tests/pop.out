> no such key

> ok

> nil

> ok

> ok

> [123 44 b]

> 123

> [44 b]

> 44

> [b]

> b

> []

> nil

> []

> [12]

> bye
